library/ix-dev/enterprise/minio/README.md

```diff
@@ -0,0 +1,16 @@
+# MinIO
+
+[MinIO](https://min.io) is a High Performance Object Storage released under Apache License v2.0.
+It is API compatible with Amazon S3 cloud storage service. Use MinIO to build high performance infrastructure
+for machine learning, analytics and application data workloads.
+
+> During the installation process, a container will be launched with **root** privileges. This is required
+> in order to apply the correct permissions to the MinIO data directory. Afterward, the `MinIO` container
+> will run as a **non**-root user (`568`).
+> Same applies to the `postgres` container. This will run afterwards as a **non**-root user (`999`).
+> On each upgrade, a container will be launched with **root** privileges in order to apply the correct
```

Comment:
Not really, but because this is a backup dir we should be sure that we can write to it.
Probably apply the logic we did on qbit? Check parent dir before chowning?